core/src/main/scala/kafka/server/ConfigHandler.scala

```diff
@@ -109,14 +105,14 @@ class QuotaConfigHandler(private val quotaManagers: QuotaManagers) {
 
   def updateQuotaConfig(sanitizedUser: Option[String], clientId: Option[String], config: Properties) {
     val producerQuota =
-      if (config.containsKey(QuotaConfigOverride.ProducerOverride))
-        Some(new Quota(config.getProperty(QuotaConfigOverride.ProducerOverride).toLong, true))
+      if (config.containsKey(DynamicConfigs.Client.ProducerOverrideProp))
+        Some(new Quota(config.getProperty(DynamicConfigs.Client.ProducerOverrideProp).toLong, true))
```

Comment:
I think it would be better to call the quota properties `ProducerByteRateOverrideProp` and `ConsumerByteRateOverrideProp` since they are no longer associated with a Quota-specific class.